javascript/beniatlas.js

```diff
@@ -786,42 +786,51 @@ $(document).ready(function() {
                     //BUTTON LOGIC//
 ////////////////////////////////////////////////////
 
+    $('#map').on('click', function(){
```

Comment:
The code in here runs whenever the users click anywhere on the map.

Then, it checks for selected extents and zooms.

Code taken from here:
http://comments.gmane.org/gmane.comp.gis.openlayers.devel.ol3/2902